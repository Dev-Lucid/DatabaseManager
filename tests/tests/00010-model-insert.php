<?php
$out = '';
$table = dbm::model('table1');
$table['table1_vc1'] = 'test';
$table->save();

$out .= 'new id: '.$table['table1_id']."\n";

$table->new_query()->filter('table1_vc1','test');
$out .= $table->dump(false);

file_put_contents($output_path,$out);
?>