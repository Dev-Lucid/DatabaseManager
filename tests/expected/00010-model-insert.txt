new id: 13
Array
(
    [table1_id] => 13
    [table1_vc1] => test
    [table1_vc2] => 
    [table1_i1] => 
    [__max_page] => 0
)

